src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/ServiceProviderCommentTest.java

```diff
@@ -0,0 +1,43 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.Test;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class ServiceProviderCommentTest {
```

Comment:
JUnit 5 test classes and methods don't need to be `public`. They can/should be restricted to default visibility. Please apply to this whole PR

Reply:
That's a good point! I've missed that because of the Intellij auto-generators probably. Thanks!

2485712019083f696f5463449bb0ad08372bfc29

Reply:
And the methods!
```
@Test
void testSomething() {
}
```

Reply:
Oh, yes. Here we go: 0c74445eb2abe2bda994fad7b5a699c954cc5c8e
Thanks!